common/types/errors.go

```diff
@@ -61,11 +64,15 @@ func (err DomainNotActiveError) Error() string {
 }
 
 func (err EntityNotExistsError) Error() string {
-	return fmt.Sprintf("EntityNotExistsError{Message: %v, CurrentCluster: %v, ActiveCluster: %v}",
```

Comment:
So in the thrift error type were these not nullable?

Reply:
Those were nullable int thrift. However `Error()` methods are not generated and initial commited version was simplified and not included optional printing of nil fields.